lib/voucherifyService.js

```diff
@@ -0,0 +1,127 @@
+'use strict'
+
+const _ = require('lodash')
+const config = require('config').get('voucherifyNodeJsExampleApp')
+const moment = require('moment')
+const Promise = require('bluebird')
+const voucherifyClient = require('voucherify')
+
+const applicationKeys = config.get('applicationKeys')
+
+const testVouchersCategory = config.get('testVouchersCategory')
+const numberOfSuppliedTestVouchers = config.get('numberOfSuppliedTestVouchers')
+const email = process.env.EMAIL || config.get('email')
+const applicationId = process.env.APPLICATION_ID || applicationKeys.applicationId
```

Comment:
Those const values are used just in one place in line 18. Thus I suggest that we don't create them just use specific conf values in the places where they needed.